### PR TITLE
Misc/5397 remove language flags and update documentation

### PR DESCRIPTION
**Pull request by gbonaspetti**
#5397 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend/doc/i18n.md [hunk 1]
```diff
@@ -36,7 +31,7 @@ An untranslated key leads to an English string appearing in the Tracim interface
 
 #### The key appears in the `.json` file
 
-Do step a) and b) in the corresponding `.json` file, in folder `i18next.scanner`.
+As the errors you can make suggestions, or modify them directly if you have an account, on our [Weblate](https://hosted.weblate.org/projects/tracim/).
```
Comment (Relf-Algoo) [inline frontend/doc/i18n.md:34]:
```suggestion
Same as the errors : you can make suggestions, or modify them directly if you have an account, on our [Weblate](https://hosted.weblate.org/projects/tracim/).
```

File: frontend/doc/i18n.md [hunk 2]
```diff
@@ -69,4 +64,4 @@ export default translate()(MyComponent)
 
 IV) You can destruct `t` from `this.props` in the `render()` like it is done in most components.
 
-V) Do steps 3), 4), 5)
+V) Do steps 3), 4)
```
Comment (Relf-Algoo) [inline frontend/doc/i18n.md:67]:
```suggestion
V) Follow steps 3), 4)
```

**Review by Relf-Algoo — CHANGES_REQUESTED** [review]
I suggested two changes. (I don't know if I did it correctly)

File: frontend/doc/i18n.md [hunk 3]
```diff
@@ -36,7 +31,7 @@ An untranslated key leads to an English string appearing in the Tracim interface
 
 #### The key appears in the `.json` file
 
-Do step a) and b) in the corresponding `.json` file, in folder `i18next.scanner`.
+Same as the errors : you can make suggestions, or modify them directly if you have an account, on our [Weblate](https://hosted.weblate.org/projects/tracim/).
```
Comment (Haskaris) [inline frontend/doc/i18n.md:34]:
```suggestion
Same as the errors: you can make suggestions, or modify them directly if you have an account, on our [Weblate](https://hosted.weblate.org/projects/tracim/).
```

**Review by Haskaris — APPROVED** [review]

**Comment by inkhey** [comment]
tested on branch

**Review by Relf-Algoo — APPROVED** [review]

**Pull request merged**